jujugui/static/gui/src/app/components/budget-table/row/test-row.js

```diff
@@ -133,7 +133,7 @@ describe('BudgetTableRow', function() {
               </div>
               {undefined}
             </div>
-            {undefined}
+            <div className="twelve-col no-margin-bottom" />
```

Comment:
Adding a conditional would remove this empty div.. And all the ones int he tests that follow.